src/styles/blocks/calendar.scss

```
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  justify-content: center;
+
+  &__month {
+    display: flex;
+    gap: $gap-between-days;
+    flex-wrap: wrap;
+    max-width: calendarWidth(
+      $day-size,
+      $gap-between-days,
+      $calendar-side-paddings
+    );
+    padding: 20px $calendar-side-paddings;
+
+    @for $index from 28 through 30 {
+      &--month-length--#{$index} :nth-child(n+#{$index + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__day {
+    position: relative;
+    @include squere(100px);
+    border: 1px solid #000;
```

Create a variables for colors
```suggestion
    border: 1px solid $c-black;
```